cibuild.sh

```diff
@@ -48,6 +48,14 @@ function python-tools {
   PYTHONUSERBASE=$prebuilt/pylocal
   export PYTHONUSERBASE
   add_path $PYTHONUSERBASE/bin
+  case $os in
+    Darwin)
+      PIP=pip3
```

Comment:
Should we always use pip3?

Reply:
maybe. i just don't want to break linux.

Reply:
we can find the regression quickly with ci, so let's keep the code clean as much as we can.

Reply:
ok

Reply:
The latest apache nightly build server build break without pip3 installed : (
./cibuild.sh: line 214: pip3: command not found

Reply:
I wonder if they have python3 installed at all (we should since python 2 is EOL). We could try |python3 -m pip` for pip3.

Do we have any information on what the build machine is built with?

Reply:
I see that it would be fairly trivial to have it use our docker container to the build if that would be easier for us to maintain

Reply:
Just issue a jira to ask infra team to help: https://issues.apache.org/jira/browse/INFRA-20237.
There is still parallel build issue which could only be reproduced in apache build server with more powerful cpus.  Once it's fixed, I think use github docker container to run the nightly build would be good.

Reply:
I'm not suggesting removing it, I'm just saying I can change the Jenkins config to run the container instead. It would still run the parallel job we just would have better control of the software on the host. 

I don't have a strong opinion just wanted people to know it is an option

Reply:
@btashton  Sorry for my misunderstanding. Glad to hear that you could help to change Jenkins config to run the contianer. There used to be several times asking for infra team to install new tools in need. This could make it more easy to maintain.